reaper/reaper.go

```diff
@@ -139,6 +139,8 @@ func processContainerJSON(ctx context.Context, container types.ContainerJSON, do
 		return fmt.Errorf("Could not determine whether or not process with exe %q / and stat %v was a titus executor", link, stat)
 	}
 
+	checkIfFuseWedgedPidNs(container.State.Pid, taskID)
```

Comment:
shouldn't this go as a `else` on line 124?
(we only want to run this if the pid exists)

Reply:
`err` could be other things there (EINVAL maybe if things are really screwed up), we check for these non-ENOENT errors on line 126 and early return, so I think this is only called when the pid exists (and when the executor pid exists).